docs/Concepts/Proof-of-Stake.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Ethereum proof of stake
+---
+
+# Proof of stake
+
+[The Merge](Merge.md) transitions Ethereum Mainnet to
+[proof of stake (PoS)](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) consensus.
+
+In Ethereum's PoS, you must run a [full node](Merge.md#execution-and-consensus-clients) and
+[stake 32 ETH](https://ethereum.org/en/staking/) to become a validator.
+
+!!! important
+
+    Withdrawing staked ETH isn't yet supported and will be included in a separate upgrade following The Merge.
+
+The PoS mechanism randomly chooses validators to propose or validate blocks on the
+[Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) (consensus blocks) in defined time frames (slots).
+
+Proposers are responsible for proposing new consensus blocks, and non-proposing validators are responsible for
+validating (attesting to) proposed blocks.
+Validators are rewarded for proposing and attesting to consensus blocks eventually included in the Beacon Chain, and
+their stake is slashed if they fail to validate or if they attest to malicious blocks, incentivizing good behavior.
```

Comment:
You are not slashed for failing to validate - only for specific malicious behaviour.  Being slashed is a large reduction in staked ETH and the validator is forcibly exited so is no longer a validator.  Being penalised is a small reduction in ETH (approximately equal to what you would have earned) and the validator continues being an active validator.

The distinction is important because a lot of people are confused by it and then setup complex systems to avoid downtime that then winds up getting them slashed.  It's actually much better to miss an attestation or block than to get slashed.